src/DynamoCoreWpf/Properties/Resources.en-US.resx

```diff
@@ -760,7 +760,7 @@
     <comment>Start page | Link to videos</comment>
   </data>
   <data name="StartPageVisitDynamoBim" xml:space="preserve">
-    <value>Visit www.dynamobim.org</value>
+    <value>Visit website</value>
```

Comment:
Visit {0} website, so that we can insert product name while making a call. Visit website, sounds imcomplete.